src/Angor/Client/Pages/Signatures.razor

```diff
@@ -175,19 +175,36 @@
         await SignService.LookupInvestmentRequestsAsync(project.ProjectInfo.NostrPubKey, null,// project.LastRequestForSignaturesTime , async 
             (investorPubKey,encryptedMessage, requestTime) =>
         {
-            Logger.LogDebug($"Event received");
+            var timeArrived = requestTime;
```

Comment:
Why do you need an extra property? you can just change the name of request time if you don't like it.

Reply:
readability I suppose, the requestTime is the name of the property on the method signature, ok I will change that